### DISCUSSION TITLE
Repositories for Windows Users

### DISCUSSION THREAD

**Discussion by jahansen777** [openssl/openssl#23206] [Q&A] [answered]
Sorry for the noob question, are there any windows distributions for openssl, or are they all for linux? All I see in the latest release are tar balls and source code, no zips or extraction packages? Do I need to look somewhere else?

**Comment by levitte** [comment]
OpenSSL currently only ships source code tarballs, which can be built on Windows just as well as on Linux (and a plethora of other platforms).

Binary packages are currently left to the diverse platform vendors (for Linux, that's the diverse Linux distros), or 3rd parties.  For Windows, I know of two such 3rd parties:

- chocolatey: https://community.chocolatey.org/packages/openssl
- Shining Light: https://slproweb.com/products/Win32OpenSSL.html

**Comment by mattcaswell** [comment, marked as answer]
We maintain a list of third party supplied OpenSSL binaries here:

https://wiki.openssl.org/index.php/Binaries

The list includes a number of different options for Windows.
